src/fl.go

```diff
@@ -156,13 +152,7 @@ func main() {
 
 	// Otherwise check for TUI flag
 	if Flags.Tui {
-		/* @DISABLED while changing argparse and adding config options
-		 * err = ui.RunProgram(&Flags)
-		 * if err != nil {
-		 * 	fmt.Printf("Error running TUI: %v", err)
-		 * 	os.Exit(1)
-		 * }
-		 */
+		startTui(Flags, Config)
```

Comment:
i'd add the comment that it's disabled here, not in start tui, i'd leave that function operational
you can also look at go generate to compile in or out the tui code, something you can look at if youre interested